### PR TITLE
CP-18170: Set and remove PVS-proxy rules live

### PR DESCRIPTION
**Pull request by robhoes**

### CODE REVIEW

**Comment by robhoes** [comment]
Depends on xenserver/xcp-idl#6.

**Comment by robhoes** [comment]
@lindig Could you review this please?

File: ocaml/xapi/pvs_proxy_control.ml [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+		farm_name = farm_rc.API.pVS_farm_name;
+		servers;
+		clients;
+		vdi;
+	}
+
+(** Request xcp-networkd to update a farm's PVS-proxy daemon configuration,
+ *  for all locally running proxies, taking into account starting and stopping proxies *)
+let update_farm_on_localhost ~__context ~farm ~vdi ?(starting_proxies=[]) ?(stopping_proxies=[]) () =
+	debug "Updating PVS farm %s. Starting proxies: [%s]. Stopping proxies: [%s]"
```
Comment (lindig) [inline ocaml/xapi/pvs_proxy_control.ml]: For consistency, you could add a `.` at the end of the string.

**Comment by lindig** [comment]
Again, this looks really solid. I can't rule out that I'm missing some logic but the code is well crafted.

File: ocaml/xapi/pvs_proxy_control.ml (hunk 1 above)
Comment (robhoes) [inline ocaml/xapi/pvs_proxy_control.ml]: Updated.

**Comment by robhoes** [comment]
@lindig Thanks for the quick review(s)!

**Pull request merged**